weblate/templates/weblate_auth/user_list.html

```diff
@@ -22,6 +22,4 @@ <h4>{{ object }}</h4>
 {% endfor %}
 </div>
 
-{% include "paginator.html" %}
```

Comment:
I think the paginator should stay here. Again, the results are still paginated here, you've removed only the UI.